tests/docker/universum_test_env/Dockerfile

```diff
@@ -4,6 +4,15 @@ ARG PYTHON=python3.7
 # Update package list and install wget
 RUN apt update && apt install -y wget software-properties-common
 
+# Install python 2 for pylint testing
+RUN DEBIAN_FRONTEND=noninteractive apt install -y python
+RUN wget --no-check-certificate -O get-pip.py 'https://bootstrap.pypa.io/pip/2.7/get-pip.py'
+RUN python get-pip.py
+RUN python -m pip install pylint
+
+# Install uncrustify for testing
+RUN DEBIAN_FRONTEND=noninteractive apt install -y uncrustify
+
```

Comment:
Please move this block to the end of the file to use existing python layers (that are common for several different docker images)

Reply:
moving python2 installation after python3 installation causes some issues